docs/how_to_ignore.md

```diff
@@ -1,4 +1,12 @@
-# How to ignore certain violations in _flake8_ mode
+# How to ignore certain violations
```

Comment:
@jsh9 I am removing this `in _flake8_ mode` because the `index.md` doesn't have it, and this page is now talking about both possible routes

Reply:
Got it.  Yeah this sounds good